src/notation/internal/notationinteraction.cpp

```diff
@@ -1150,8 +1153,27 @@ void NotationInteraction::startDragCopy(const EngravingItem* element, QObject* d
         m_drag = nullptr;
     });
 
-    static QPixmap pixmap(2, 2); // null or 1x1 crashes on Linux under ChromeOS?!
+    const qreal adjustedRatio = 0.4;
```

Comment:
I'm not completely confident about having this "magic number" here. I assume you found this value empirically. Maybe you should use `configuration()->notationScaling()`, which returns the scaling that would be needed to achieve 100% zoom in the score view; that might work well on more different displays with different settings. 
Anyway, that will probably become clear during testing; for now let's leave this value here and see if it produces correct results for everyone.

Reply:
Actually I got that value from this code
```
    const qreal spatium = configuration()->paletteSpatium();
    const qreal sizeRatio = spatium / gpaletteScore->style().spatium();
    const qreal adjustedRatio = sizeRatio * 1.5; // Yeah 1.5 used was here was empirically
```
taken from `PaletteCellIconEngine` but wasn't sure if we could use it here as I thought we are getting `configuration()` from `INJECT_STATIC(IPaletteConfiguration, configuration)` so I thought it must be something palette specific because of `IPaletteConfiguration`